app/src/main/java/com/github/swent/echo/data/repository/datasources/FileCache.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.swent.echo.data.repository.datasources
+
+/** Interface for file caching. */
+interface FileCache {
+
+    /**
+     * Get the file content by name.
+     *
+     * @param name The name of the file to get.
```

Comment:
It could be useful to add a `@return` section in this comment to specify when this function shoud return `null`